mne/io/eeglab/eeglab.py

```diff
@@ -0,0 +1,374 @@
+        The time interval to apply baseline correction.
+        If None do not apply it. If baseline is (a, b)
+        the interval is between "a (s)" and "b (s)".
+        If a is None the beginning of the data is used
+        and if b is None then b is set to the end of the interval.
+        If baseline is equal to (None, None) all the time
+        interval is used.
+        The baseline (a, b) includes both endpoints, i.e. all
+        timepoints t such that a <= t <= b.
+    reject : dict | None
+        Rejection parameters based on peak-to-peak amplitude.
+        Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg'.
+        If reject is None then no rejection is done. Example::
+
+            reject = dict(grad=4000e-13, # T / m (gradiometers)
+                          mag=4e-12, # T (magnetometers)
+                          eeg=40e-6, # uV (EEG channels)
+                          eog=250e-6 # uV (EOG channels)
+                          )
+    flat : dict | None
+        Rejection parameters based on flatness of signal.
+        Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg', and values
+        are floats that set the minimum acceptable peak-to-peak amplitude.
+        If flat is None then no rejection is done.
+    reject_tmin : scalar | None
+        Start of the time window used to reject epochs (with the default None,
+        the window will start with tmin).
+    reject_tmax : scalar | None
+        End of the time window used to reject epochs (with the default None,
+        the window will end with tmax).
+    montage : str | None | instance of montage
+        Path or instance of montage containing electrode positions.
+        If None, sensor locations are (0,0,0). See the documentation of
+        :func:`mne.channels.read_montage` for more information.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Notes
+    -----
+    .. versionadded:: 0.11.0
+
+    See Also
+    --------
+    mne.Epochs : Documentation of attribute and methods.
+    """
+    @verbose
+    def __init__(self, input_fname, events=None, event_id=None, tmin=0,
+                 baseline=None,  reject=None, flat=None, reject_tmin=None,
+                 reject_tmax=None, montage=None, verbose=None):
+        from scipy import io
+        eeg = io.loadmat(input_fname, struct_as_record=False,
+                         squeeze_me=True)['EEG']
+
+        if events is None and eeg.trials > 1:
+            # first extract the events and construct an event_id dict
+            event_name, event_latencies, unique_ev = [], [], []
+            for ep in eeg.epoch:
+                if not isinstance(ep.eventtype, string_types):
+                    event_type = '/'.join(ep.eventtype.tolist())
+                    event_name.append(event_type)
+                    # store latency of only first event
+                    event_latencies.append(ep.eventurevent[0])
```

Comment:
I  do not know what specific columns of MNE python `events` array mean, but EEGLAB `urevents` are not latencies. For given event the `urevent` value informs wchich "original event" it corresponds to. The "original events" are the events that were present in the signal before any manipulations that result in loss of events (epoching, epoch removal etc.). Is this what you store in the first column of `events` array?